templates/partials/armour-location.html

```diff
@@ -30,12 +30,12 @@
                 {{/if}}
 
                 {{#if item.Qualities.length}}
-                <span class="quality label">Q: </span>
+                <span class="quality label">{{localize "Q:"}}</span>
                 {{{array item.Qualities "text armour-quality"}}}
                 {{/if}}
 
                 {{#if item.Flaws.length}}
-                <span class="flaw label">F:</span>
+                <span class="flaw label">{{localize "F:"}}</span>
```

Comment:
As with the other ones, I don't see the corresponding keys for this. I feel like the key should be more specific too.